tests/foreman/destructive/test_katello_certs_check.py

```diff
@@ -117,6 +117,7 @@ def test_positive_install_sat_with_katello_certs(certs_vm_setup):
     result = rhel_vm.subscription_manager_attach_pool([settings.subscription.rhn_poolid])[0]
     for repo in getattr(constants, f"OHSNAP_RHEL{version}_REPOS"):
         rhel_vm.enable_repo(repo, force=True)
+    # What is the purpose of this?
```

Comment:
@omkarkhatavkar this is a question for you, if you don't mind filling me in on the details.